macros/scripts/events.asm

```diff
@@ -315,15 +315,15 @@ giveegg: MACRO
 	db \2 ; level
 ENDM
 
-	enum givepokeitem_command ; $2f
-givepokeitem: MACRO
-	db givepokeitem_command
+	enum givepokemail_command ; $2f
```

Comment:
Please add `givepokeitem` and `checkpokeitem` to macros/legacy.asm since they're renamed. Also update docs/event_commands.md, and rename `Script_givepokeitem` and `Script_checkpokeitem` in engine/overworld/scripting.asm to match.

Reply:
Fixed.